book-service/src/main/java/com/jpop/bookservice/service/BookService.java

```diff
@@ -0,0 +1,31 @@
+package com.jpop.bookservice.service;
+
+import com.jpop.bookservice.model.Book;
+import com.jpop.bookservice.repository.BookRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class BookService {
+    @Autowired
+    BookRepository bookRepository;
+
+    public List<Book> findAll() {
+        return bookRepository.findAll();
+    }
+
+    public Optional<Book> findById(Long id) {
+        return bookRepository.findById(id);
+    }
+
+    public Book save(Book book) {
+        return bookRepository.save(book);
+    }
+
+    public void deleteById(Long id) {
+        bookRepository.deleteById(id);
+    }
+}
```

Comment:
You may handle exceptions in a common place.
May be Generic Exception Handler class